### PR TITLE
Konstantinos /  [FE Deriv.com squad] Location description for Deriv (BVI) 

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/careers/_model/_locations/_locations.ts [hunk 1]
```diff
@@ -522,6 +523,31 @@ export const georgetown: LocationsType = {
...
+export const roadtown: LocationsType = {
+    name: 'roadtown',
+    job_location: 'British-Virgin-Islands',
+    continent: 'caribbean',
+    display_continent: 'Caribbean',
+    link: '/careers/locations/roadtown/',
+    display_name: 'Road Town',
+    display_thumbnail_name: 'Road Town',
+    display_title_name: 'Road Town',
+    img_alt: ' Deriv’s Cayman Islands office at Road Town ',
```
Comment (mitra-deriv) [inline src/pages/careers/_model/_locations/_locations.ts:535]: let's change the `image_alt` per SEO suggestion we having Cayman and Road town both here which they are separate office 🙏

File: src/pages/careers/_model/_locations/_locations.ts [hunk 2]
```diff
@@ -522,6 +523,31 @@ export const georgetown: LocationsType = {
...
+    map_office_name: '(BVI)',
+    country: 'British Virgin Islands',
+    flagIcon: CaymanFlagIcon,
```
Comment (mitra-deriv) [inline src/pages/careers/_model/_locations/_locations.ts:538]:
```suggestion
    flagIcon: BVIFlagIcon,
```

File: src/pages/careers/_model/_locations/_locations.ts [hunk 3]
```diff
@@ -522,6 +523,31 @@ export const georgetown: LocationsType = {
...
+    thumbnail: 'thumbnail_roadtown',
+    first_p:
+        'Experts in regulatory and legislative requirements will work together at Deriv (BVI) to provide compliance guidance in our British Virgin Islands licensed operations. Their expertise in corporate law, anti-money laundering, and other legislation will enable them to anticipate and mitigate regulatory risks. ',
+    first_img: 'living_in_roadtown',
+    first_img_alt: 'Deriv’s office in the Cayman Islands',
```
Comment (mitra-deriv) [inline src/pages/careers/_model/_locations/_locations.ts:543]: ditto, this is from Cayman Islands office should be in respective of the BVI(Road Town) office

**Review by mitra-deriv — CHANGES_REQUESTED** [review]

File: src/pages/careers/locations/roadtown/index.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        }
+    }
+`
+
+const Roadtown = () => {
+    const images = useStaticQuery(query)
+
+    return (
+        <Layout type="careers">
+            {/* ---------WILL FILL THIS SECTION ONCE SEO IS DONE------
```
Comment (habib-deriv) [inline src/pages/careers/locations/roadtown/index.tsx:35]:
```suggestion
          Done keep this code commented. If needed, please speak with Swati put the right content here
```

Comment (habib-deriv) [inline src/pages/careers/locations/roadtown/index.tsx:35]: Don't keep this SEO part commented out, Please speak with Swati if needed on this

Comment (habib-deriv) [inline src/pages/careers/locations/roadtown/index.tsx:35]: Don't keep this SEO part commented out, Please speak with Swati if needed on this

**Review by habib-deriv — CHANGES_REQUESTED** [review]

File: src/pages/careers/_model/_locations/_locations.ts [hunk 5]
```diff
@@ -23,6 +23,7 @@ import CyprusFlagIcon from 'images/svg/careers/flag_cyprus.svg'
 import CaymanFlagIcon from 'images/svg/careers/flag_cayman.svg'
 import RwandaFlagIcon from 'images/svg/careers/flag_rwanda.svg'
 import BelarusFlagIcon from 'images/svg/careers/flag_belarus.svg'
+import BVIFlagIcon from 'images/svg/careers/flag_france.svg'
```
[the remark this answers is not in the document]
Comment (konstantinosG-deriv) [reply src/pages/careers/_model/_locations/_locations.ts:26]: Oh I had these correct but most likely I hit undo and forgot to change them back. Thanks

File: src/pages/careers/locations/roadtown/index.tsx (hunk 4 above)
Comment (konstantinosG-deriv) [reply src/pages/careers/locations/roadtown/index.tsx:35]: Swati is already working on it and will add it when it's done, I thought we could review code for now

Comment (hubert-deriv) [reply src/pages/careers/locations/roadtown/index.tsx:35]: SEO changes for this are being done in a different PR?

If that is the case, perhaps it's possible to test and merge the SEO changes together with your changes in one PR?

Comment (konstantinosG-deriv) [reply src/pages/careers/locations/roadtown/index.tsx:35]: SEO changes are done on another card in redmine with a pdf file or doc attached.

**Review by NikitK-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**